.lagoon.yml

```diff
@@ -9,3 +9,19 @@ tasks:
             . scripts/jira-post-comment.sh $JIRA_ENDPOINT $JIRA_USERNAME $JIRA_PASSWORD $LAGOON_GIT_BRANCH sdpa $LAGOON_GIT_BRANCH "$LAGOON_PR_TITLE"
           fi
         service: storybook
+
+environments:
+  master:
+    routes:
+      - storybook:
+        - "www.ripple.sdp.vic.gov.au":
+            tls-acme: 'true'
+            insecure: Allow
+            hsts: max-age=31536000
+        - "ripple.sdp.vic.gov.au":
+            tls-acme: 'true'
+            insecure: Allow
+            hsts: max-age=31536000
+    monitoring_urls:
+      - "www.ripple.sdp.vic.gov.au"
```

Comment:
@conantran , we shouldn't have the www version in here. Please raise a ticket for Mark to fix this so www redirects to the root domain.

Reply:
Changes pushed.